### ISSUE TITLE
Allow adding more sections to the specific page (e.g. Group, People and more)

### ISSUE THREAD

**Issue by GezimSejdiu** [SmartDataAnalytics/sda.tech#18]
Me as one of the members of the SDA website should be able to extend the section on my profile page for example or for Groups as well : 

- [ ] Allow users adding new sections (like teaching, thesis topic opened by them, Reseach interest, awards, presentation or talks and more, one example: http://sda.cs.uni-bonn.de/people/gezim-sejdiu/ )  to the People page by just adding new section type on RDD and render it via ttl file.

**Comment by RohanAsmat** [comment]
I am not sure if it can be exactly achieved as if we go with the current rdf-model, there should be a limit from a development perspective to know the possibility of max sections and their content inorder to display them appropriately based on the content. As for the sake of general user it could be anything.  Or I think we might want to reconsider our current rdf-model that we can extend the group and person classes with section type class so that it can add support to dynamic sections.

**Comment by GezimSejdiu** [comment]
@NargisTahara until we find out the solution on how to add sections based on the user preferences, we at least should list thesis topics (like we do for courses) on the user profile. The idea is that in case I want to do a thesis with someone from SDA (e.g. we are offering a lab at uni.bonn for BigDataAnalytics and students want to see if Hajira or I are offering topics). As of now, students should go on the [thesis topics page](http://sda.tech/masterthesis/#) (which is good to have the list for general purpose -- when the student is approaching everyone at SDA) but in case they know the domain and the people they can go directly from there to my profile and see all the topics offered at SDA supervised by me. 

It could be placed as a sub-section under Teaching to the user profile (http://sda.tech/Person/GezimSejdiu/ ).

best,

**Comment by NargisTahara** [comment]
@GezimSejdiu  Do we need to add separate sections for the bachelor, master and on Going thesis on user profile?

**Comment by GezimSejdiu** [comment]
No, I think we should list only the titles of the thesis and maybe mention on the end that it is for a master/ or bachelor level.
 * Thesis topic 1 (M)
 * Thesis topic 2 (M; B) (in case it is dedicated to both levels).
reg. the ongoing thesis (good point. usually it will be good for each of the profiles to have a sub-section which list all their supervision thesis, but we can think about it later)
